events/FlavorSPA.txt

```diff
@@ -233,7 +233,7 @@ country_event = {
 		name = "flavor_spa.EVTOPTA3150"
 		define_advisor = {
 			type = inquisitor
-			name = "Tom�s de Torquemada"
+			name = "Tom�s de Torquemada"
```

Comment:
Just tested and in game it looks crap :( 
I don't really get these though, this is not a change I made to the code, but got committed automatically. What encoding should be used?

Reply:
I am not sure, maybe here: https://eu4.paradoxwikis.com/Localisation I don't know what it's supposed to look like either.

Reply:
my VSC does not see that this is a difference. However the code on master works in game
![image](https://i.imgur.com/5fWdPiu.png)
Branch code spews some gibberish instead of Spanish a', not really sure how to fix it yet, probably revert this file (and others if needed) and edit it manually on GH site

Reply:
The wiki states that localization files: "The file must be encoded as UTF-8 with a byte order mark (UTF-8-BOM)." I don't know if its the case here.

Reply:
Yeah this is a good lead, it was just UTF-8. Pursuing this line.

Reply:
There is a couple of places this happens, not just here.

Reply:
While localization files are encoded with UTF-8 with BOM, the same is not the case for the code .txt files. After half a day of failing at solving this, a clue on how to finally fix this was found in this Reddit thread:
https://www.reddit.com/r/eu4/comments/2s7nk3/modding_does_anybody_know_how_to_type_all_the/

I encoded the FlavorSPA.txt with ISO 8859-1, saved, tested (first time something worked as intended) and pushed.